test/test_tensordict.py

```diff
@@ -541,11 +541,6 @@ def test_to_tensordict(self, td_name, device):
     def test_select(self, td_name, device, strict, inplace):
         torch.manual_seed(1)
         td = getattr(self, td_name)(device)
-        if td_name == "autonested_td":
-            pytest.skip(
-                "Select function is not yet designed for auto-nesting case."
-                " Skipping auto-nesting test case!!"
-            )
```

Comment:
I thought we hadn't adapted `select` yet, but this test is passing?

Reply:
Now seems yes